lib/connection.ex

```diff
@@ -128,7 +129,10 @@ defmodule Janus.Connection do
   # Callbacks
 
   @impl true
-  def init({transport_module, transport_args, handler_module, handler_args}) do
+  def init(
+        {transport_module, transport_args, handler_module, handler_args},
+        cleanup_interval \\ @cleanup_interval
```

Comment:
1. Why this change needed to happen?
2. If that change is going to stay I would either
a) change the name
```suggestion
        cleanup_interval \\ @default_cleanup_interval
```
Just because it makes much more sense semantically.
b) put that thing into config and read that in init. The reason being if this is going to be changed I guess it's going to be done through configuration, and not through a code change.

Reply:
After brief conversation we came to conclusion that tuple argument should be replaced with map (or consider record) and to apply 2.a scenario.